### PR TITLE
[FEATURE Collect] Disable price filter + 'bid' filter combination

### PR DESCRIPTION
**Pull request by yuki24**
finishes https://artsyproduct.atlassian.net/browse/GROW-927

This is functional but without tests. I'll look to see if I could add some more test coverage, but feel free to merge if this PR is good enough.

## Screenshot

kapture 2018-10-08 at 16 06 48

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 5210fce79ddb4bf83bb9b4d897168b6928dc98bd

https://deploy-preview-1381--artsy-reaction.netlify.com

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx [hunk 1]
```diff
@@ -54,6 +54,7 @@ class Filter extends Component<Props, State> {
         max={FilterState.MAX_PRICE}
         step={50}
         defaultValue={[initialMin, initialMax]}
+        disabled={filters.isWayToBuySelected()}
```
Comment (l2succes) [inline src/Apps/Collect/Components/ArtworkGrid/index.tsx]: we should only disable if `this.state.at_auction` is `true`

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx [hunk 2]
```diff
@@ -90,13 +93,15 @@ class Filter extends Component<Props, State> {
             onSelect={value => {
               return filters.setFilter("acquireable", value, mediator)
             }}
+            disabled={priceSelected}
```
Comment (l2succes) [inline src/Apps/Collect/Components/ArtworkGrid/index.tsx]: see above, only for `Bid`

File: src/Apps/Collect/Components/ArtworkGrid/index.tsx (hunk 1 above)
Comment (yuki24) [reply src/Apps/Collect/Components/ArtworkGrid/index.tsx]: Good catch :+1:

**Pull request merged**